src/Controls/src/Core/Button/Button.iOS.cs

```diff
@@ -477,5 +480,11 @@ private protected override void OnHandlerChangingCore(HandlerChangingEventArgs a
 			_originalImageSize = CGSize.Empty;
 			_originalCGImage = null;
 		}
+
+		private static void MapFormatting(IButtonHandler handler, IText button)
+		{
+			// Update all of the attributed text formatting properties
+			handler.PlatformView?.UpdateCharacterSpacing(button);
```

Comment:
Should this static method be removed and we just fire the character spacing handler on like 469?

Reply:
Yes, this makes sense! Done :)